src/content/docs/apm/agents/go-agent/configuration/distributed-tracing-go-agent.mdx

```diff
@@ -111,25 +105,33 @@ For more help finding your traces in the UI:
 * [Understand and use the distributed tracing UI](/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui)
 * [Query distributed trace data](/docs/understand-dependencies/distributed-tracing/ui-data/query-distributed-trace-data)
 
-### Set up Infinite Tracing [#infinite-tracing]
+  </Step>
+</Steps>
+
+## Infinite Tracing (optional) [#infinite-tracing]
 
 Standard distributed tracing for APM agents ([above](#quick-start-apm)) captures up to 10% of your traces, but if you want us to analyze all your data and find the most relevant traces, you can set up Infinite Tracing.
```

Comment:
I'd remove the `above` link and added to the `Standard distributed tracing for APM agents` text.
I'd removed the tip add the link directly to `Infinite Tracing`.

Reply:
@nbaenam I fixed this!